cms/check_rule_counts

```diff
@@ -21,23 +21,59 @@
 """
 Probe to check rules.
 """
-from __future__ import print_function
 
 import datetime
 import sys
 import traceback
+import json
+
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
 
 from rucio.core import monitor
 from rucio.db.sqla import models
 from rucio.db.sqla.constants import (RuleState)
 from rucio.db.sqla.session import get_session
 from rucio.db.sqla.util import get_count
+from rucio.common.config import config_get
+
 from sqlalchemy import func
 
+PROM_SERVERS = config_get('monitor', 'prometheus_servers', raise_exception=False, default='')
+if PROM_SERVERS != '':
+    PROM_SERVERS = PROM_SERVERS.split(',')
+else:
+    PROM_SERVERS = None
+
+prom_labels_config = config_get('monitor', 'prometheus_labels', raise_exception=False, default='{}')
+extra_prom_labels = json.loads(prom_labels_config)
+
 # Exit statuses
 OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
 
 if __name__ == '__main__':
+    registry = CollectorRegistry()
+
+    labelnames = ['state']
+    not_ok_labels = ['Not_ok_rules']
+    stuck_cnt_labels = ['Stuck_cnt']
+    replicating_cnt_labels = ['Replicating_cnt']
+
+
+    labelnames.extend(extra_prom_labels.keys())
+    not_ok_labels.extend(extra_prom_labels.keys())
+    stuck_cnt_labels.extend(extra_prom_labels.keys())
+    replicating_cnt_labels.extend(extra_prom_labels.keys())
+
+
+    rules_count_gauge = Gauge('rucio_judge_rules_state_count', 'Number of rules in a given state',
```

Comment:
This was just rucio_rules_count. Please keep the metric names the same. So what is here with period substituted with underscore and then with whatever mapping/labeling you find in https://github.com/FernandoGarzon/CMSKubernetes/blob/master/kubernetes/rucio/statsd-prometheus-mapping.yaml

Reply:
I left it as rucio_rules_coutn, but still, I think I'll have to modify the YAML file.